python/cuml/ensemble/randomforest_common.pyx

```diff
@@ -235,13 +236,16 @@ class BaseRandomForestModel(Base):
             if y_dtype != np.int32:
                 raise TypeError("The labels `y` need to be of dtype"
                                 " `int32`")
-            self.classes_ = rmm_cupy_ary(cp.unique, y_m)
-            self.num_classes = len(self.classes_)
+            temp_classes = cp.unique(y_m)
```

Comment:
This changes is removing part of the exposed API. Is there a reason we don't want to expose `self.classes_`, which enhances compatibility with scikit-learn APIs?